Scripts/DCS-BIOS/lib/common/Logger.lua

```diff
@@ -236,4 +235,43 @@ function Logger:log_table_indexes(tab)
 	end
 end
 
+--- Logs an array
+--- @require array array to log
+function Logger:log_array(array)
```

Comment:
Shouldn't this have a log level that is passed through from like log_info to log_error?

Reply:
This was thought to be a debug function